<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Page3adv</class>
 <widget class="QDialog" name="Page3adv">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create VDF Metadata File: Advanced Options</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>48</x>
     <y>67</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Storage Block Factor</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="bsxSpinner">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>70</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Internal storage blocking factor expressed in grid points (NXxNYxNZ).  Default: 64x64x64.</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>64</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="bszSpinner">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Internal storage blocking factor expressed in grid points (NXxNYxNZ).  Default: 64x64x64.</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>64</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="periodicxButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>110</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Axis periodicity specifies whether data will &quot;loop back&quot; to its start or end point, after exceeding the bounds of an axis.</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>54</x>
     <y>110</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Axis Periodicity</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>37</x>
     <y>24</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Compression Ratio List</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="compressionRatioBox">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>26</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Colon delimited list compression ratios. The default is 1:10:100:500.  The maximum compression ratio is wavelet and block size dependent. </string>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Lucida Grande'; font-size:13pt;&quot;&gt;1:10:100:500&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="periodicyButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="bsySpinner">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>51</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Internal storage blocking factor expressed in grid points (NXxNYxNZ).  Default: 64x64x64.</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="value">
    <number>64</number>
   </property>
  </widget>
  <widget class="QCheckBox" name="periodiczButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Z</string>
   </property>
  </widget>
  <widget class="QPushButton" name="acceptButton">
   <property name="geometry">
    <rect>
     <x>187</x>
     <y>172</y>
     <width>114</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>57</x>
     <y>172</y>
     <width>114</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="restoreDefaultButton">
   <property name="geometry">
    <rect>
     <x>96</x>
     <y>142</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Restore Default Values</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
